[
  {
    "url": "https://api.github.com/repos/electron/electron/issues/comments/108723895",
    "html_url": "https://github.com/electron/electron/issues/1858#issuecomment-108723895",
    "issue_url": "https://api.github.com/repos/electron/electron/issues/1858",
    "id": 108723895,
    "user": {
      "login": "hokein",
      "id": 2557445,
      "avatar_url": "https://avatars2.githubusercontent.com/u/2557445?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/hokein",
      "html_url": "https://github.com/hokein",
      "followers_url": "https://api.github.com/users/hokein/followers",
      "following_url": "https://api.github.com/users/hokein/following{/other_user}",
      "gists_url": "https://api.github.com/users/hokein/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/hokein/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/hokein/subscriptions",
      "organizations_url": "https://api.github.com/users/hokein/orgs",
      "repos_url": "https://api.github.com/users/hokein/repos",
      "events_url": "https://api.github.com/users/hokein/events{/privacy}",
      "received_events_url": "https://api.github.com/users/hokein/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2015-06-04T05:05:48Z",
    "updated_at": "2015-06-04T05:05:48Z",
    "body": "Looks like you don't have permission. here might be the answer for you: http://stackoverflow.com/questions/2009278/python-multiprocessing-permission-denied\n"
  },
  {
    "url": "https://api.github.com/repos/electron/electron/issues/comments/108909599",
    "html_url": "https://github.com/electron/electron/issues/1858#issuecomment-108909599",
    "issue_url": "https://api.github.com/repos/electron/electron/issues/1858",
    "id": 108909599,
    "user": {
      "login": "apm",
      "id": 38468,
      "avatar_url": "https://avatars2.githubusercontent.com/u/38468?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/apm",
      "html_url": "https://github.com/apm",
      "followers_url": "https://api.github.com/users/apm/followers",
      "following_url": "https://api.github.com/users/apm/following{/other_user}",
      "gists_url": "https://api.github.com/users/apm/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/apm/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/apm/subscriptions",
      "organizations_url": "https://api.github.com/users/apm/orgs",
      "repos_url": "https://api.github.com/users/apm/repos",
      "events_url": "https://api.github.com/users/apm/events{/privacy}",
      "received_events_url": "https://api.github.com/users/apm/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2015-06-04T14:08:07Z",
    "updated_at": "2015-06-04T14:08:07Z",
    "body": "Aha, thanks. I rebooted that machine for the first time in a while, and permissions must have reverted to the broken state.\n"
  }
]